React/Modules/RCTTiming.m

```diff
@@ -171,28 +171,35 @@ - (dispatch_queue_t)methodQueue
   return RCTJSThread;
 }
 
+// Called from JS Thread.
```

Comment:
I am concerned that it's always true.

Reply:
🤔 For the old module system, seems we use this rule to do things, for example : https://github.com/facebook/react-native/blob/master/React/Modules/RCTUIManager.m#L97-L101, https://github.com/facebook/react-native/blob/ac7ec4602fd4d8ec033a084df6a92437ea2d3a8a/React/Modules/RCTAlertManager.m#L42 .... , but yeah TM may change it which call `invalidate` always on `JS Thread`. I'll add an `dispatch` here. :)